packages/core/src/babelPlugins/__tests__/babel-plugin-redwood-directory-named-imports.ts

```diff
@@ -1,45 +1,53 @@
 import * as babel from '@babel/core'
+import { ensurePosixPath } from '@redwoodjs/internal'
 
 import namedDirectory from '../babel-plugin-redwood-directory-named-import'
 
+// So tests pass on Windows too
+const POSIX_DIRNAME = ensurePosixPath(__dirname)
+
 const testCases = [
-  // Directory named exports
+  // Directory named imports
   {
     input: 'import pew from "./__fixtures__/directory-named-imports/Module"',
-    output:
-      'import pew from "./__fixtures__/directory-named-imports/Module/Module"',
+    output: `import pew from "${POSIX_DIRNAME}/__fixtures__/directory-named-imports/Module/Module.js";`,
```

Comment:
Had a thought. Something feels off here, because imports in NodeJS are always in unix style. I guess it's because of the way that we get the absolute paths.

Reply:
Not exactly happy with this. There's nothing actionable, just thought I would mention it.